ProductsWebApi/ProductsWebApi/Models/Products.cs

```diff
@@ -1,6 +1,6 @@
 ﻿namespace ProductsWebApi.Models
 {
-    public class Product
+    public class Products
```

Comment:
Can keep it as Product. 
The properties you have inside should talk with the class name
Eg. Name is for a Product, not for Products.
Know what i mean?

Reply:
Changed from 'products' to 'product'